rusk-recovery/src/keys.rs

```diff
@@ -123,7 +120,7 @@ pub fn run_circuit_keys_checks(
 }
 
 pub fn exec(keep_keys: bool) -> Result<(), Box<dyn std::error::Error>> {
-    lazy_static::initialize(&PUB_PARAMS);
+    Lazy::force(&PUB_PARAMS);
```

Comment:
Is this really necessary?

Reply:
Weird...
Just noticed that without this the crs is never checked (and therefore never cached)